libr/core/disasm.c

```diff
@@ -1096,6 +1096,12 @@ static void ds_begin_line(RDisasmState *ds) {
 	if (ds->pj) {
 		pj_o (ds->pj);
 		pj_kn (ds->pj, "offset", ds->vat);
+		if (ds->analop.jump != UT64_MAX ) {
+			pj_kN (ds->pj, "jump", ds->analop.jump);
```

Comment:
why the jump is signed and the fail unsigned?

Reply:
use pj_kn instead of pj_kN

Reply:
Because it's a PoC and I don't know the API. Thanks I will fix it.